### ISSUE TITLE
Make chapter “jump to” use always the ascending order

### ISSUE THREAD

**Issue by Palloxin** [LNReader/lnreader#1149]
### Describe your suggested feature

All details here #815 due to wrong tag.
TLDR: if you set decrescent sorting and try to teleport(jump) to chapter 200 in an entry with 1000 chaps you will teleport to chapter 800.
I believe it's completely wrong and confusing and it should jump/teleport only based on ascending order.
If i type 10 it should "jump to" chapter 10, always.

### Other details

Related issues that need a force sort order to function properly:
#957
#815
#752
#1149

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/LNReader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[2.0.0](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
